.circleci/config.yml

```diff
@@ -147,8 +147,6 @@ workflows:
             - lint-css
             - lint-l10n
             - deploy:
-                requires:
-                    - lint-js
```

Comment:
@mansaj Could you clarify why this was necessary? Shouldn't this task have succeeded?

Reply:
@Vinnl Good question, so this took me a while to figure out 

The reason we need to get rid of this requirement is that lint-js actually does NOT run in tagged workflow. In CircleCI, tags only run when they are specified (see deploy step below). Because lint-js wasn't running, the deploy step did not run either, resulting in our circleCI skipping deploy completely, thus skipping our Jenkins pipeline.

Reply:
Ah gotcha, and so the reason it broke recently is because #3004 introduced 212fe07, which added this dependency. Thanks!